packages/caver-klay/caver-klay-kct/contract/token/KIP17/KIP17MetadataMintable.sol

```diff
@@ -0,0 +1,46 @@
+pragma solidity ^0.5.0;
+
+import "./KIP17Metadata.sol";
+import "../../access/roles/MinterRole.sol";
+import "../../introspection/KIP13.sol";
+
+
+/**
+ * @title KIP17MetadataMintable
+ * @dev KIP17 minting logic with metadata.
```

Comment:
I think you should separate KIP17Mintable and KIP17Metadata.

Reply:
When user do mint, tokenURI which is a function defined in metadata will be set.
That's why I think i can't separate mint and metatdata.

Because metadata is optional, anyone who does not implement it should implement mintable seperately, not metadataMintable. 
But if the user implements metatdata and mintable tokens, I think mintable should be implemented with metadata.

If you think name is strange, i can change it.
I think we need to discuss the application of metaMintable to KIP17.

Reply:
Or what do you think about implementing both mintable and metadataMintable?

Reply:
Okay. I will do that in KIP-17.